### DISCUSSION TITLE
震惊！！！神秘录像店竟做此事！！！

### DISCUSSION THREAD

**Discussion by ybbgh** [share121/inter-knot#2997] [其他]
朋友们，最近六分街那家录像店RandomPlay进了很多和龙有关联的录像带！
不要觉得这没什么！最近他们的员工都有些疲惫，昨天顾客也少了很多！
鄙人对山水和卦象略有研究，依我看...
RandomPlay是要吸干六分街的龙脉以此振兴啊！！！

**Comment by waerte-yang** [comment]
哦？展开说说！（细说）

**Reply by ybbgh** [reply]
我自学了一点老祖宗的堪舆风水之术，发现这其中大有文章!
先说这进的录像带，基本都是与龙相关，啧啧，你品，你细品。
再说进的录像带的时长（叠个甲先，没看过录像带，朋友说的)都是1小时14分钟左右
有个名目叫「凿龙眼」，是借运冲煞、绝户刨根的禁术!
我算是明白了，总有些人见不得六分街繁荣，要让六分街的人过不好日子!

**Comment by 3673237414** [comment]
![mmexport1724030416974](https://github.com/user-attachments/assets/a4af34be-2ca4-4fd9-9eb3-6552a98bfb59)
不会是这个龙吧……

**Reply by ybbgh** [reply]
唉！你怎么这样！请严肃看待这件事！这关乎六分街的命运，有人不见得六分街繁荣！

**Comment by HyperMuteki0347** [comment]
怪不得，我前两天去店里租带子，刚进去就感觉头晕晕的，回家大吐一阵实在受不了倒厕所里睡了23小时！普通人一去那里都会被吸干的吧...好可怕！

**Reply by ybbgh** [reply]
竟然已经有人中招了吗？朋友现在还好吗？那家录像店太邪恶了！

**Comment by pkgfedgjjtd** [comment]
我有一个朋友去了，结果回来就一直呕吐不止

**Reply by ybbgh** [reply]
可恶啊！这么多人中招了，我们一起去把那家录像店举报给治安局

**Comment by RaphaelHider** [comment]
录像店最近的动作就很怪，其实零号空洞附近有个危机应对的斯科特哨站，我经常看到店主被防卫队偷偷送到哨站，这种级别的地方他一个开录像店的有什么资格去啊？我怀疑他们就是是内外勾结搞破坏的。
还有一天夜里我看见一个鼠系人进RandomPlay，没一会拿了个录像带出来，我觉得可疑就跟上去了，结果发现他从辉映广场的一个隐蔽入口进入了空洞，吓得我赶紧溜了。如果是正经人为什么要走这种通道？我觉得那盘录像带肯定是用来在空洞里害人的。
一件可疑的事可能是巧合，两件三件放在一起还能叫巧合么？加上楼主的信息我觉得这店主肯定在搞什么阴谋，真的别怪我多想。

**Reply by aaajszhjzbsjhzh** [reply]
我嘞个花火，来新艾利都找乐子来了

**Reply by ybbgh** [reply]
什么？！居然还和调查协会有关系，这我们怎么应对，他背后有人，完了。

**Reply by ybbgh** [reply]
呵呵呵...你要记住，什么是愚蠢，什么时候该装傻

**Reply by ybbgh** [reply]
晚上别睡得太死哦
